docs/multicluster/user-guide.md

```diff
@@ -75,8 +75,7 @@ To deploy Multi-cluster Controller in a dual-role cluster, please refer to
 
   ```bash
   kubectl create ns antrea-multicluster
-  curl -L https://github.com/antrea-io/antrea/releases/download/$TAG/antrea-multicluster-leader-namespaced.yml >   antrea-multicluster-leader-namespaced.yml
-  sed 's/changeme/antrea-multicluster/g' antrea-multicluster-leader-namespaced.yml | kubectl apply -f -
+  kubectl apply -f https://github.com/antrea-io/antrea/releases/download/$TAG/antrea-multicluster-leader-namespaced.yml >   antrea-multicluster-leader-namespaced.yml
```

Comment:
Have we talked somewhere that "antrea-multicluster" needs to be changed in the YAML to use another NS?

Reply:
Let me check, probably need to add a few descriptions for this.

Reply:
Done, add a new section to highlight how to change default leader Namespace.